docs/example-workflow.rst

```diff
@@ -3,17 +3,161 @@
+    ==>     root experiment_index: 3
+    ==>     log file: /home/jovyan/benchpark/wkp/kripke-benchmark/tutorial/workspace/logs/setup.2025-07-09_18.08.23/kripke.kripke.kripke_kripke_single_node_strong_scaling_caliper_time_mpi_4_2_2_64_64_32_64_1_128_128_4_16.out
+    ==>   Returning to log file: /home/jovyan/benchpark/wkp/kripke-benchmark/tutorial/workspace/logs/setup.2025-07-09_18.08.23.out
+    ==> Experiment #4 (4/4):
+    ==>     name: kripke.kripke.kripke_kripke_single_node_strong_scaling_caliper_time_mpi_4_4_2_64_64_32_64_1_128_128_4_32
+    ==>     root experiment_index: 4
+    ==>     log file: /home/jovyan/benchpark/wkp/kripke-benchmark/tutorial/workspace/logs/setup.2025-07-09_18.08.23/kripke.kripke.kripke_kripke_single_node_strong_scaling_caliper_time_mpi_4_4_2_64_64_32_64_1_128_128_4_32.out
+    ==>   Returning to log file: /home/jovyan/benchpark/wkp/kripke-benchmark/tutorial/workspace/logs/setup.2025-07-09_18.08.23.out
+
+-----------------------------------------
+Step 6: Run Kripke Experiments with Flux
+-----------------------------------------
+
+**Run** the Kripke experiments, which will launch jobs through the `Flux resource manager <https://flux-framework.org/>`_::
+
+    ramble --disable-progress-bar --workspace-dir /home/jovyan/benchpark/wkp/kripke-benchmark/tutorial/workspace on
+
+This should return something like this on the terminal after running the command::
+
+    ==> Streaming details to log:
+    ==>   /home/jovyan/benchpark/wkp/kripke-benchmark/tutorial/workspace/logs/execute.2025-07-09_18.14.08.out
+    ==>   Executing 4 out of 4 experiments:
+    ==>   Log files for experiments are stored in: /home/jovyan/benchpark/wkp/kripke-benchmark/tutorial/workspace/logs/execute.2025-07-09_18.14.08
+    ==> Running executors...
+    ƒV54uD5o5  
+    ƒV57fKkEK  
+    ƒV5ANUS6s  
+    ƒV5D498h5
+
+.. note::
+
+    You can check on the progress of your runs by running the command::
+
+        flux jobs [-a]
+
+    The -a is optional; adding it to the command will show all the jobs submitted including jobs that are currently running and completed jobs.
+
+
+The experiments will run sequentially, and the total time to complete all four experiments should be 8-9 minutes. Upon completion, each
+experiment will generate a caliper file. When all the experiments are finished, running the command :code:`flux jobs -a` will show all the
+jobs. A completed job will be green with ``CD`` as its status (Example below):
+
+    .. image:: ./finished_experiments_example.png
+       :alt: Example output of flux jobs -a when experiments are finished
+       :width: 750px
+       :align: center
+
+------------------------
+Step 7: Analyze Results
+------------------------
+
+1. **Conduct** pre-defined analysis with Benchpark::
+
+    benchpark analyze --workspace-dir /home/jovyan/benchpark/wkp/kripke-benchmark/tutorial/workspace
```

Comment:
@stephanielam3211  Try:
1. `--no-mpi`
2. `--no-mpi and --top-n-regions 10`
3. You can try `--chart-fontsize` to change chart fontsize